crates/superposition/Superposition.cac.toml

```diff
@@ -23,6 +21,6 @@ tenant = { schema = { "type" = "string", "enum" = ["test", "dev"] } }
 
 [context."$tenant == 'dev'"]
 mandatory_dimensions = []
-experiments_webhook_config = { "enabled" = false, "configuration" = { "url" = "http://localhost:8080/config/test", "method" = "Get", "headers" = [
-    "ConfigVersion",
+experiments_webhook_config = { "enabled" = false, "configuration" = { "url" = "http://localhost:8080/config/test", "method" = "Get", "service_headers" = [
```

Comment:
Shouldn't you define custom_headers here or in default_config?

Reply:
Yeah i'll define it
But i have kept it not required, so would be fine to leave it also